src/js/characters.ts

```diff
@@ -0,0 +1,378 @@
+const mesia_patch_data = Array.from(atob(mesia_spritesheet_chr), c => c.charCodeAt(0))
```

Comment:
atob is deprecated, TextEncoder/TextDecoder is the preferred solution now.  Hopefully this gets cleaned up, tho, so I don't really care too much - that's mainly FYI.